.github/workflows/connector-ci-tests.yml

```diff
@@ -70,6 +70,8 @@ jobs:
     runs-on: linux-24.04-large # Custom runner, defined in GitHub org settings
     env:
       GCP_GSM_CREDENTIALS: ${{ secrets.GCP_GSM_CREDENTIALS }}
+      GCP_PROJECT_ID: ${{ secrets.GCP_PROJECT_ID || vars.GCP_PROJECT_ID }}
```

Comment:
Generally, these are non-secret, but this can be a clear-text variable or a secret, according to the user's preference. If not set in the repo, will default to our Airbyte-internal project by the CDK CLI.